other/materials_designer/definitions.py

```diff
@@ -0,0 +1,103 @@
+from IPython.display import display, Javascript
+import json
+
+
+def submit(materials):
```

Comment:
(1) Maybe we should use the `get/set` naming - `get_materials` and `set_materials` here?
(2) Let's add dosctrings, pls

Reply:
If this is popular convention -- Ok